### PR TITLE
N2 Locker Additions

### PR DESCRIPTION
**Pull request by falcon2346**
[content]
## What this does
This adds a bit of variety to the contents of N2 lockers, similar to standard emergency lockers. N2 lockers now have a 50% chance to carry 2 large red tanks and 2 masks, as normal, however they also have a 30% chance to contain a Vox First Aid Kit (detailed below), a 10% chance to have one red tank replaced by an extended capacity N2 tank, and a 10% chance to contain both an extended capacity tank and a first aid kit. While the chance of a first aid kit is slightly higher than for O2 lockers (a total of 40% as opposed to 35%) this is offset by the vastly lower number of N2 lockers. For instance, Box contains only 4 N2 lockers across the entire map, so in an average round will contain fewer than 2 Vox First Aid Kits.

Vox First Aid Kits contain the following items: 2 dexalin pills, 1 priaxate autoinjector, 2 nitrogen syringes, and either a cracker or a fake stack of cash. When played with the cash plays the same sound as when you deposit money into your PDA.

dreamseeker_bGoWJ5LiKd
dreamseeker_1VIqGGZdTS

This also adds a new chem, priaxate, which currently only exists in the new autoinjectors, but can be made by combining gravy and tricordrazine. Functionally, this is a slightly weaker version of doctor's delight, but for Vox is slightly more powerful for healing oxygen and toxin damage. For reference, A standard Autoinjector heals 24 of each damage type. A priaxate injector will instead heal 18 of each, though for Vox will instead heal 30 oxygen and toxin damage in addition to 18 brute and burn.

## Why it's good
Adds some more variety to N2 lockers. I honestly don't think I've ever seen one get opened outside of rounds where a lot of people get forcibly turned into Vox.

## How it was tested
Spawned a bunch of N2 lockers, everything seemed to spawn at the right rate. Successfully mixed priaxate.

## Changelog

:cl:
 * rscadd: Added Vox first aid kits and a new chem, priaxate.
 * tweak: Adjusted emergency N2 lockers to have randomized contents.

### CODE REVIEW

**Comment by Eneocho** [comment]
on my way to steal the turkizol injectors and recycle them for cash

**Comment by SECBATON-GRIFFON** [comment]
not sure i get why the fake cash is in these but i like the rest a lot

**Comment by falcon2346** [comment]
> not sure i get why the fake cash is in these but i like the rest a lot

Cash has a psychological healing factor for birds.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -106,6 +106,25 @@
 	item_state = "firstaid-internalbleed"
 	items_to_spawn = list(/obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector = 4)
 
+/obj/item/weapon/storage/firstaid/vox
+	name = "oxygen deprivation first-aid kit"
```
Comment (SonixApache) [inline code/game/objects/items/weapons/storage/firstaid.dm:110]:
```suggestion
	name = "nitrogen deprivation first-aid kit"
```

**Comment by SonixApache** [comment]
i still think turkizol is a bad name and making gravy a component will allow people to access gravy without needing traders which is also bad
muh star trek chem naming conventions

File: code/game/objects/items/weapons/storage/firstaid.dm (hunk 1 above)
Comment (SonixApache) [inline code/game/objects/items/weapons/storage/firstaid.dm]: alternatively, 
```suggestion
	name = "oxygen exposure first-aid kit"
```

Comment (falcon2346) [reply code/game/objects/items/weapons/storage/firstaid.dm]: Oops, I don't know how I missed that. I like this suggestion more

**Comment by SonixApache** [comment]
👍

**Comment by hacker-on-steroids** [comment]
what if instead of a new chem this was moved to lexorin, since it deals oxygen damage

File: code/modules/reagents/reagents/reagents_medical.dm [hunk 2]
```diff
@@ -1690,6 +1690,40 @@ var/global/list/charcoal_doesnt_remove=list(
...
+	name = "Priaxate"
+	id = PRIAXATE
+	description = "Priaxate is a broad spectrum medication formulated for the unique biology of vox. While still effective on other species, some of its effects are less potent."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#CD8471" //rgb: 205, 132, 113
+	density = 1.29
+	specheatcap = 0.72
+	custom_metabolism = 0.4
+
+/datum/reagent/turkizol/on_mob_life(var/mob/living/M)
```
Comment (hacker-on-steroids) [inline code/modules/reagents/reagents/reagents_medical.dm]: did you mean priaxate here?
i don't think turkizol exists

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -139,6 +139,20 @@
...
 
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/priaxate
+	name = "priaxate injector"
+	desc = "A rapid and safe way to administer small amounts of drugs by untrained or trained personnel. More effective on vox."
+	icon_state = "turkey1"
+	item_state = "turkey"
+	flags = FPRINT
+	refill_reagent_list = list(PRIAXATE = 5)
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/turkizol/update_icon()
```
Comment (hacker-on-steroids) [inline code/modules/reagents/reagent_containers/hypospray.dm]: did you switch turkizol and priaxate?
i'm surprised this even compiles

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 4]
```diff
@@ -106,6 +106,25 @@
...
 	items_to_spawn = list(/obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector = 4)
 
+/obj/item/weapon/storage/firstaid/vox
+	name = "oxygen exposure first-aid kit"
+	desc = "A box full of vox medical essentials."
+	icon_state = "vox"
+	item_state = "firstaid-advanced"
+	items_to_spawn = list(
+		/obj/item/weapon/reagent_containers/pill/dexalin = 2,
+		/obj/item/weapon/reagent_containers/hypospray/autoinjector/turkizol,
```
Comment (hacker-on-steroids) [inline code/game/objects/items/weapons/storage/firstaid.dm]: ?

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 3 above)
Comment (hacker-on-steroids) [reply code/modules/reagents/reagent_containers/hypospray.dm]: ah it seems you did in https://github.com/vgstation-coders/vgstation13/pull/37142/commits/525f56caa4df405e7acaf5749ac57f9947bb9361 but didn't fully replace everything

Comment (falcon2346) [reply code/modules/reagents/reagent_containers/hypospray.dm]: Thought I got everything, I'll give it another look through. My b

**Comment by falcon2346** [comment]
Okay, it should all be correct now.

**Comment by SECBATON-GRIFFON** [comment]
priaxate will be a fun addition to heal pills

File: code/modules/reagents/reagents/reagents_medical.dm [hunk 5]
```diff
@@ -1690,6 +1690,40 @@ var/global/list/charcoal_doesnt_remove=list(
...
+	if(M.getOxyLoss())
+		M.adjustOxyLoss(-3 * REM)
+	if(M.getBruteLoss())
+		M.heal_organ_damage(3 * REM, 0)
+	if(M.getFireLoss())
+		M.heal_organ_damage(0, 3 * REM)
+	if(M.getToxLoss())
+		M.adjustToxLoss(-3 * REM)
+
+/datum/reagent/priaxate/on_mob_life(var/mob/living/M, var/alien)
```
Comment (jwhitak) [inline code/modules/reagents/reagents/reagents_medical.dm]: Duplicate proc. Merge this one with the one above it.

**Review by jwhitak — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagents/reagents_medical.dm (hunk 5 above)
Comment (falcon2346) [reply code/modules/reagents/reagents/reagents_medical.dm]: Done

**Review by jwhitak — APPROVED** [review]

**Pull request merged**